frontend/src/components/student/QuestionNumber.tsx

```diff
@@ -23,13 +23,15 @@ const QuestionNumber = ({
           fontSize="20px"
           icon={<CheckmarkIcon />}
           isActive
-          minWidth="3.7rem"
+          maxWidth="3.6rem"
+          minWidth="3.6rem"
           onClick={onClick}
           variant="primary"
         />
       ) : (
         <Button
-          minWidth="3.7rem"
+          maxWidth="3.6rem"
```

Comment:
Just plain `width` doesn't work right

Reply:
yea, bc we need to override this [line](https://github.com/uwblueprint/jump-math/blob/staging/frontend/src/themes/components/button.ts#L23). alternatively, could make a new size variant but not sure its necessary